doc/source/examples/unresolved-cfd-dem/boycott-effect/boycott-effect.rst

```diff
@@ -0,0 +1,372 @@
+   	set grad div = true
+   	set void fraction time derivative = false
+   	set drag force = true
+   	set buoyancy force = true
+   	set shear force = true
+   	set pressure force = true
+   	set drag model = difelice
+   	set post processing = true
+   	set coupling frequency = 250
+   	set implicit stabilization = false
+   	set grad-div length scale = 0.005
+   	set vans model = modelA
+    end
+    
+We determine the drag model to be used for the calculation of particle-fluid forces. We enable buoyancy, drag, shear and pressure forces. For drag, we use the Difelice model to determine the momentum transfer exchange coefficient. The VANS model we are solving is model A. Other possible option is model B.
+
+Finally, the linear and non-linear solver controls are defined.
+
+Non-linear solver control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    subsection non-linear solver
+        set solver = inexact_newton
+   	set tolerance               = 1e-8
+  	set max iterations          = 10
+  	set verbosity               = verbose
+  	set matrix tolerance        = 0.75
+    end
+
+We use the inexact_newton solver as to avoid the reconstruction of the system matrix at each Newton iteration. For more information about the non-linear solver, please refere to the `Non Linear Solver Section <../../../parameters/cfd/non-linear_solver_control.html>`_
+    
+Linear solver control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    subsection linear solver
+        set method                                 = gmres
+  	set max iters                              = 1000
+  	set relative residual                      = 1e-3
+  	set minimum residual                       = 1e-10
+  	set ilu preconditioner fill                = 0
+  	set ilu preconditioner absolute tolerance  = 1e-12
+  	set ilu preconditioner relative tolerance  = 1
+  	set verbosity               = verbose
+  	set max krylov vectors = 200
+    end
+
+For more information about the non-linear solver, please refere to the `Linear Solver Section <../../../parameters/cfd/linear_solver_control.html>`_
+
+Running the CFD-DEM simulation
+------------------------------
+
+The simulation is run using the cfd_dem_coupling_3d application as per the following command:
+
+.. code-block:: text
+
+    path_to_cfd_dem_application/cfd_dem_coupling_3d boycott-effect.prm 
+
+Results
+--------
+
+The results are shown in an animation below. The sedimentation of the particles in a vertical and inclined tube demonstrate different behaviors. This clearly shows the boycott effect as the fluid circulates in the inclined tube resulting in a larger velocity for both the fluid and particles. Thus, the particles fall further compared to the vertical tube where the fluid velocity is almost null and the particles' acceleration is low. 
+
+.. raw:: html
+
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/DNoiy35ha8k" frameborder="0" allowfullscreen></iframe>
```

Comment:
Can you make the animation with more frames and smoother? it's quite unclear right now (it could be made much smoother and pleasant to the eye :) )

Reply:
done